.github/workflows/test.yml

```diff
@@ -10,30 +8,38 @@ on:
     branches:
       - main
 
-# Set the language, install dependencies, and run the tests
+env:
+  NPY_PROMOTION_STATE: weak_and_warn
```

Comment:
When running pytest with `NPY_PROMOTION_STATE: weak_and_warn`,  I get an overflow error for Numpy < 2. Presumably some of the tests will fail here too, if Numpy < 2 is in the Numpy matrix.

Reply:
The job "Test / Python 3.9 / ubuntu-latest / numpy==1.26.4" should test this.